experimental/null_safety/null_safe_app/lib/badapp.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
```

Comment:
All the Dart files need copyright headers. Should be able to use the same one as you'll find in the other apps.

Reply:
Yes, of course